src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/AbstractAppSchemaMockData.java

```diff
@@ -172,6 +183,15 @@ public AbstractAppSchemaMockData(Map<String, String> namespaces) {
         styles = new File(data, "styles");
         styles.mkdir();
 
+        appschema = new File(data, "appschema");
+        appschema.mkdir();
+
+        appschemagml3132 = new File(appschema, "appschema-gml3132");
+        appschemagml3132.mkdir();
+
+        appschemacache = new File(appschemagml3132, "app-schema-cache");
+        appschemacache.mkdir();
+
```

Comment:
I guess these directories are for the mappings and xsd in the resource folder. Please check the comment below.

Reply:
fixed.